l10n_multi_country/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module will be used as a base for others in which it is necessary to use it to hide some fields that need to be added and are only necessary if the company is from country X; for example, a l10n-italy module that adds fields to the contact (https://github.com/OCA/l10n-italy/blob/12.0/l10n_it_fatturapa/models/partner.py#L13-L59) but that are only required when the company is Italian.
```

Comment:
Instead of https://github.com/OCA/l10n-italy/blob/12.0/l10n_it_fatturapa/models/partner.py#L13-L59 please use a permalink https://github.com/OCA/l10n-italy/blob/f01ef9c96319b45ac8e777e338476b60f67ac75a/l10n_it_fatturapa/models/partner.py#L13-L59